src/services/projects.js

```diff
@@ -420,14 +421,19 @@ module.exports = class ProjectsHelper {
 	 * @returns {JSON} - List of reviewers from the org
 	 */
 
-	static async reviewerList(organization_id, pageNo, limit) {
+	static async reviewerList(user_id, organization_id, pageNo, limit) {
 		try {
-			const reviewers = await userRequests.list(common.REVIEWER, pageNo, limit, '', organization_id)
+			let reviewers = await userRequests.list(common.REVIEWER, pageNo, limit, '', organization_id)
+			const userList = reviewers.data.result.data.filter((user) => user.id !== user_id)
+
 			if (reviewers.success) {
 				return responses.successResponse({
 					statusCode: httpStatusCode.ok,
 					message: 'REVIEWER_LIST_FETCHED_SUCCESSFULLY',
-					result: reviewers.data.result,
+					result: {
+						data: userList,
+						count: reviewers.data.result.count,
```

Comment:
@adithyadinesh0412  if user is removed then count also will change right

Reply:
Can we do this in user service? because this logic will be a little tricky to do here. 
Can we have this as an enhancement in user service search call where we can pass the userIds to be ignored.
@priyanka-TL , @aks30